StyleGuide.md

```diff
@@ -1070,6 +1120,26 @@ Use the following HTML and Liquid to include an image in a Markdown page:
 
 Example: [Decommission Nodes](https://www.cockroachlabs.com/docs/stable/remove-nodes.html#step-1-check-the-node-before-decommissioning)
 
+<a name="videos"></a>
+
+### Videos
+
+Like images, use videos to clarify a topic, but only use them as needed. Typically, videos should be hosted on the official [CockroachDB YouTube page](https://www.youtube.com/@cockroachdb).
```

Comment:
Should we add something here about videos usually being surfaced by Marketing?

Reply:
Done!